public/locales/en/translation.json

```
@@ -299,7 +299,10 @@
       "secrets": {
         "list": {
           "title": "Secrets",
-          "empty": "No secrets exist yet"
+          "empty": "No secrets exist yet",
+          "createBtn": "Create Secret",
+          "searchPlaceholder": "Filter Secrets...",
```

```suggestion
          "createBtn": "Create secret",
          "searchPlaceholder": "Filter secrets...",
```
For consistency, I'll now always go for "only first letter capitalized" unless we decide otherwise (already did this in the last PR as well).